backend/server.go

```diff
@@ -13,14 +15,14 @@ const PORT = 5001 // todo: put into environment var
 
 func main() {
 	r := chi.NewRouter()
+	config.Connect()
 	r.Mount("/api", api())
-
+	r.Mount("/api/users", routes.UsersRoute())
```

Comment:
Can we add this to the func below (`func api() chi.Router)`? So that all our API routes are already prefixed with `/api` instead of needing to specify it each time.